src/compas_fab/robots/wrench.py

```diff
@@ -0,0 +1,353 @@
+		"""
+		Discretize the wrench
+		"""
+		binary_ft = [0,0,0,0,0,0]
+
+		#forces
+		for idx, w in enumerate(wrench[0:3]):
+			if (w > forces_threshold): binary_ft[idx] = 1
+			elif (w < -forces_threshold): binary_ft[idx] = -1
+			else: binary_ft[idx] = 0
+		#torques
+		for idx, w in enumerate(wrench[3:6]):
+			if (w > torques_threshold): binary_ft[idx+3] = 1
+			elif (w < -torques_threshold): binary_ft[idx+3] = -1
+			else: binary_ft[idx+3] = 0
+		
+		return Wrench(binary_ft)
+	
+	# ==========================================================================
+	# operators
+	# ==========================================================================
+
+	def __mul__(self, n):
+		"""Create a ``Wrench`` multiplied by the given factor.
+
+		Parameters
+		----------
+		n : float
+			The multiplication factor.
+
+		Returns
+		-------
+		Wrench
+			The resulting new ``Wrench``.
+
+		"""
+		return Wrench([
+			n * self.wrench[0],
+			n * self.wrench[1],
+			n * self.wrench[2],
+			n * self.wrench[3],
+			n * self.wrench[4],
+			n * self.wrench[5] ])
+	
+	def __sub__(self, other):
+		"""Return a ``Wrench`` that is the the difference between this ``Wrench`` and another wrench.
+
+		Parameters
+		----------
+		other : wrench
+			The wrench to subtract.
+
+		Returns
+		-------
+		Wrench
+			The resulting new ``Wrench``.
+		"""
+		return Wrench([
+			self.wrench[0] - other[0],
+			self.wrench[1] - other[1],
+			self.wrench[2] - other[2],
+			self.wrench[3] - other[3],
+			self.wrench[4] - other[4],
+			self.wrench[5] - other[5] ])
+	
+	def __neg__(self):
+		"""Return the negated ``Wrench``.
+
+		Returns
+		-------
+		Wrench
+			The negated ``Wrench``.
+		"""
+		return Wrench([
+			-1.0 * self.wrench[0],
+			-1.0 * self.wrench[1],
+			-1.0 * self.wrench[2],
+			-1.0 * self.wrench[3],
+			-1.0 * self.wrench[4],
+			-1.0 * self.wrench[5] ])
+
+	def __len__(self):
+		return len(self.wrench)
+	
+	def __getitem__(self, item):
+		return self.wrench[item]
+	
+	def __eq__(self, other):
+		return np.allclose(self.wrench, other)
+
+	# ==========================================================================
+	# transformations
+	# ==========================================================================
+
+	def rotated(self, rotation_matrix):
+		"""Rotate a wrench or force-torque vector given a rotation matrix.
+
+		Parameters
+		----------
+		rotation_matrix: matrix, 3x3 rotation matrix [[xx, yx, zx], [xy, yy, zy], [xz, yz, zz]]
+
+		Returns
+		-------
+		Wrench
+			rotated wrench.
+		"""
+		rotated_wrench = []
+		for i in range(2):
+			vector = self.wrench[i * 3: i * 3 + 3]
+			# Transpose to [[x], [y], [z]]
+			vector = np.matrix(vector).T
+			# Apply rotation
+			vector = np.matmul(rotation_matrix, vector).T
+			# Convert back to vector
+			vector = vector.tolist()[0]
+			# Include in rotated_wrench
+			rotated_wrench.extend(vector)
+		return Wrench(rotated_wrench)
+
+	@property
+	def unbiased(self):
```

Comment:
Example?
`ForceTorqueSensor` method?

Reply:
removed this property in the latest commit